### PR TITLE
[aws2sqs] Add deleteQueue operation to aws2sqs component

### PR DESCRIPTION
**Pull request by VratislavHais**
When testing camel-quarkus I have stumbled upon missing operation for deleteQueue

### CODE REVIEW

File: components/camel-aws/camel-aws2-sqs/src/test/java/org/apache/camel/component/aws2/sqs/SqsProducerDeleteQueueTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package org.apache.camel.component.aws2.sqs;
```
Comment (oscerd) [inline components/camel-aws/camel-aws2-sqs/src/test/java/org/apache/camel/component/aws2/sqs/SqsProducerDeleteQueueTest.java:1]: Please add the license header

**Review by oscerd — CHANGES_REQUESTED** [review]
Just one minor finding

File: components/camel-aws/camel-aws2-sqs/src/test/java/org/apache/camel/component/aws2/sqs/SqsProducerDeleteQueueTest.java (hunk 1 above)
Comment (VratislavHais) [reply components/camel-aws/camel-aws2-sqs/src/test/java/org/apache/camel/component/aws2/sqs/SqsProducerDeleteQueueTest.java:17]: Done. Thank you for pointing it out.

**Review by oscerd — APPROVED** [review]

**Comment by oscerd** [comment]
Thanks

**Pull request merged**